### PR TITLE
feat(MeetingsAdapter): transform localShare into an object

### PR DESCRIPTION
**Pull request by karinasigartau0798**
Transform localShare into an object containing "stream" property based on MeetingStream object definition.

### CODE REVIEW

File: src/MeetingsAdapter.js [hunk 1]
```diff
@@ -16,6 +16,13 @@ import WebexAdapter from './WebexAdapter';
  * @see {@link https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack}
  */
 
+/**
+ * An object that represents a media stream.
+ *
+ * @typedef {object} MeetingStream
+ * @property {MediaStream}  stream  The media stream
```
Comment (cipak) [inline src/MeetingsAdapter.js:23]: single space before `stream`

**Comment by lalli-flores** [comment]
Can this be combined with https://github.com/webex/component-adapter-interfaces/pull/68?

**Comment by alinasuciu** [comment]
Moved the PR changes  on PR #68

**Comment by alinasuciu** [comment]
Moved the PR changes on PR #68

**Pull request closed without merging**